packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -95,22 +95,23 @@ contract PluginRepo is
         address _pluginFactory,
         bytes calldata _contentURI
     ) external auth(address(this), CREATE_VERSION_PERMISSION_ID) {
-        // Check if `_pluginFactory` is a `PluginFactoryBase` contract
+        // Check if `_pluginFactory` is a `PluginManager` contract
         if (!Address.isContract(_pluginFactory)) {
             revert InvalidContractAddress({invalidContract: _pluginFactory});
         }
-
-        try
-            PluginFactoryBase(_pluginFactory).supportsInterface(
-                PluginFactoryIDs.PLUGIN_FACTORY_INTERFACE_ID
-            )
-        returns (bool result) {
-            if (!result) {
-                revert InvalidPluginFactoryInterface({invalidPluginFactory: _pluginFactory});
-            }
-        } catch {
-            revert InvalidPluginFactoryContract({invalidPluginFactory: _pluginFactory});
-        }
+        
+        // TODO: uncommment
+        // try
+        //     PluginManager(_pluginFactory).supportsInterface(
+        //         PluginManager.PLUGIN_FACTORY_INTERFACE_ID
+        //     )
+        // returns (bool result) {
+        //     if (!result) {
+        //         revert InvalidPluginFactoryInterface({invalidPluginFactory: _pluginFactory});
+        //     }
+        // } catch {
+        //     revert InvalidPluginFactoryContract({invalidPluginFactory: _pluginFactory});
+        // }
```

Comment:
Why is this commented? Will we need it in the future?

Reply:
Will fix it soon. not a point of this PR.

Reply:
Do we have a task to track this?